### DISCUSSION TITLE
Requesting build for "ppc" cpu target

### DISCUSSION THREAD

**Discussion by Carves123** [AdguardTeam/AdGuardHome#3803] [Q&A]
I have a TP-Link WDR4900 V1.0
Hardware spec here https://openwrt.org/toh/tp-link/tl-wdr4900

I did try the MIPS variants and ARM variants and it wont run.
`line 1: syntax error: unexpected "("` - or something like that

I also tried 
`curl -s -S -L https://raw.githubusercontent.com/AdguardTeam/AdGuardHome/master/scripts/install.sh | sh -s -- -c edge`

It spits out this message:

```
starting AdGuard Home installation script
unsupported cpu type: ppc
```

Apparently wrong type of CPU.
Is there a build for ppc type cpu ?
Thanks

**Comment by ainar-g** [comment]
Hello.  Go only supports 64-bit PowerPC architectures starting with POWER8 and later, if I recall correctly, and it seems like that router has a CPU with an architecture that is older than that.

**Reply by Carves123** [reply]
https://github.com/AdguardTeam/AdGuardHome/releases/tag/v0.106.3

It is not listed here. Sorry, I'm kinna new to this.
Perhaps a little bit help please.

**Reply by ainar-g** [reply]
The programming language and compiler that we use doesn't support the CPU architecture that your router seems to have.
